### PR TITLE
[JBTM-3053] printing to debug the node identifier which is used

### PR DESCRIPTION
**Pull request by ochaloup**
for xa orphan detection filters

https://issues.jboss.org/browse/JBTM-3053

!QA_JTA !QA_JTS_JDKORB !QA_JTS_OPENJDKORB !QA_JTS_JACORB !BLACKTIE !XTS !PERF NO_WIN !RTS !TOMCAT !mysql !postgres !db2 !oracle

### CODE REVIEW

File: ArjunaJTA/jta/classes/com/arjuna/ats/jta/common/JTAEnvironmentBean.java [hunk 1]
```diff
@@ -410,6 +411,11 @@ else if(instance != oldInstance)
      */
     public void setXaRecoveryNodes(List<String> xaRecoveryNodes)
     {
+        if(jtaLogger.logger.isDebugEnabled()) {
+            jtaLogger.logger.debugf("Setting up node identifiers '%s' for which recovery"
+                + " will be performed", xaRecoveryNodes);
```
Comment (mmusgrov) [inline ArjunaJTA/jta/classes/com/arjuna/ats/jta/common/JTAEnvironmentBean.java]: Does putting a List<String> here actually print out each element of the list?

Comment (ochaloup) [reply ArjunaJTA/jta/classes/com/arjuna/ats/jta/common/JTAEnvironmentBean.java]: @mmusgrov it depends on the `toString()` method of the `List` implementation and the `ArrayList` is used here which really prints all the records. This is not true e.g. for arrays in java but from my experience it's true for all the collection java classes in the java util package.

Comment (mmusgrov) [reply ArjunaJTA/jta/classes/com/arjuna/ats/jta/common/JTAEnvironmentBean.java]: System.out.println("%s", arrayList) prints out each element as you suggest so I agree and would expect any logger should do the same.

**Comment by ochaloup** [comment]
Thanks Mike.

AS tests fails on LDAP security tests.

**Pull request merged**